packages/vow/test/watch-upgrade.test.js

```diff
@@ -29,9 +34,34 @@ test.serial('vow resolve across upgrade', async t => {
       },
     });
 
+    const watcher2 = zone.exo('DurableVowTestWatcher2', undefined, {
```

Comment:
My understanding is that the `watcher2` in this incarnation will see rejections which will be effectively unhandled for the lack of a `onRejected` method, right?

I am wondering if we should be more explicit about why we have various unhandled rejections:
- add a `onRejected` that makes a plain assertion `t.pass()` (that can be accounted for in the `t.plan()`), before rethrowing the error.
- Maybe add comments about how many time the `onRejected` is expected to be hit, and by which vows.
- Add comment when resolving each `vow` or setting up a `watch` which ones are expected to create an unhandled rejection.

Reply:
Noted at the top of the test.